Gofer.NET.Tests/GivenARedisTaskQueue.cs

```diff
@@ -40,13 +43,16 @@ public async Task ItCapturesArgumentsPassedToEnqueuedDelegate()
                 Tuple.Create<Expression<Action>, string>(
                     actionExp,
                     str);
-            
+
             // Action to expected result
             var delgates = new Tuple<Expression<Action>, string>[]
             {
                 // Exception Argument
                 TC(() => ExceptionFunc(new Exception(), semaphoreFile), new Exception().ToString()),
                 TC(() => ExceptionFunc(new CustomException(), semaphoreFile), new CustomException().ToString()),
+
+                // Cancelation Argument
+                TC(() => CancellationFunc(default, semaphoreFile), new CancellationToken().ToString()),

```

Comment:
Why pass default rather than new CancellationToken?

Reply:
I see they are effectively equivalent but I would prefer if it were consistent across the argument to CancellationFunc and the evaluation value for the semaphore. (i.e. Use default or new CancellationToken() in both spots)

Reply:
Done:)